data/json/items/armor/hats.json

```diff
@@ -321,6 +461,360 @@
+        "weight": 5
+      },
+      {
+        "id": "soccer_fan_ball",
+        "name": { "str": "soccer fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular soccer team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 1
+      },
+      {
+        "id": "basketball_fan_ball",
+        "name": { "str": "basketball fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular basketball team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 1
+      },
+      {
+        "id": "volleyball_fan_ball",
+        "name": { "str": "volleyball fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular volleyball team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 2
+      },
+      {
+        "id": "hockey_fan_ball",
+        "name": { "str": "hockey fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular hockey team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 5
+      },
+      {
+        "id": "dodgeball_fan_ball",
+        "name": { "str": "dodgeball fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular dodgeball team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 1
+      },
+      {
+        "id": "rugby_fan_ball",
+        "name": { "str": "rugby fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular rugby team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 1
+      },
+      {
+        "id": "tennis_fan_ball",
+        "name": { "str": "tennis fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular tennis team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 1
+      },
+      {
+        "id": "lacrosse_fan_ball",
+        "name": { "str": "lacrosse fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular lacrosse team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 1
+      },
+      {
+        "id": "badminton_fan_ball",
+        "name": { "str": "badminton fan trucker cap" },
+        "description": "A snapback trucker cap displaying the logo of a popular badminton team.  It shields your eyes from the sun and provides a little bit of warmth.",
+        "weight": 1
+      },
+      {
+        "id": "flatball_fan_ball",
+        "name": { "str": "flatball fan trucker cap" },
```

Comment:
May I ask what is a flatball? I was trying to translate this but my search only turned up either 1. a type of strike in golf which I think can hardly be called its own sport, or 2. fla`g`ball which is a type of football.

Reply:
Flatball is a type of table sport, kind of a kid's game but there are local clubs and surprisingly even teams of it. Flatball is not a ball, it is the name of the sport.

Reply:
In the US we usually call it disc golf. I think there's already disc golf stuff in the game.

Reply:
Thanks, but sadly I cannot find any reference on that so would you mind find one for me?

So far I've only found this (https://en.wikipedia.org/wiki/Ultimate_(sport)) which is nicknamed flatball but is obviously not a table sport.

Reply:
> Thanks, but sadly I cannot find any reference on that so would you mind find one for me?
> 
> So far I've only found this (https://en.wikipedia.org/wiki/Ultimate_(sport)) which is nicknamed flatball but is obviously not a table sport.

Hey Qrox, I'm sorry but I misunderstood it. I remade my research (the content of this PR is some years old, I just updated and PRed it) and apparently you are right, the cap is actually a Ultimate (foosball) cap, I'm sorry.